openforcefield/topology/molecule.py

```diff
@@ -469,19 +466,19 @@ def molecule_particle_index(self):
             raise ValueError('This Atom does not belong to a Molecule object')
         return self._molecule.particles.index(self)
 
-    ## From Jeff: Not sure if we actually need this
-    @property
-    def topology_atom_index(self):
-        """
-        The index of this Atom within the the list of atoms in ``Topology``.
-        Note that this can be different from ``particle_index``.
-
-        """
-        if self._topology is None:
-            raise ValueError('This Atom does not belong to a Topology object')
-        # TODO: This will be slow; can we cache this and update it only when needed?
-        #       Deleting atoms/molecules in the Topology would have to invalidate the cached index.
-        return self._topology.atoms.index(self)
+    # ## From Jeff: Not sure if we actually need this
```

Comment:
Why is this commented out? Shouldn't we delete it or add a `NotImplementedException`?

Reply:
I'm pretty sure this shouldn't be here,  but devs should discuss this - referencing #235 here.

Reply:
Removed this in 3f184a1